x-pack/test/case_api_integration/security_and_spaces/tests/common/configure/get_configure.ts

```diff
@@ -91,54 +72,6 @@ export default ({ getService }: FtrProviderContext): void => {
       expect(data).to.eql(getConfigurationOutput());
     });
 
-    it('should return a configuration with mapping', async () => {
```

Comment:
This would fail when the license is basic, so I moved this test to the trial specific tests.